tests/filter-tests.js

```diff
@@ -148,6 +148,7 @@ describe('broccoli-asset-rev', function() {
     var node = new AssetRewrite(sourcePath + '/input', {
       replaceExtensions: ['js'],
       assetMap: {
+        'images/some-image.jpg': 'images/some-image-1a2b3c4e.jpg',
         'the.map' : 'the-other-map',
         'http://absolute.com/source.map' : 'http://cdn.absolute.com/other-map'
```

Comment:
That is from https://github.com/rickharrison/broccoli-asset-rewrite/pull/8/files. I am not that familiar with source maps, but I'm sure there is a reason it was included as a test case.